api/camunda/camunda-openapi.yaml

```
@@ -3962,6 +3967,14 @@ components:
         licenseType:
           description: Will return the license type property of the Camunda license
           type: string
+        isCommercial:
+          description: Will be false when a license contains a non-commerical=true property
```

```suggestion
          description: Will be false when a license contains a non-commercial=true property
```

spelling error -- I'll fix this upstream.